agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -618,3 +619,50 @@ func TestNetworkModeBridge(t *testing.T) {
 	err := networkModeTest(t, agent, "bridge")
 	require.NoError(t, err, "Networking mode bridge testing failed")
 }
+
+// TestFluentdTag tests the fluentd logging driver option
+func TestFluentdTag(t *testing.T) {
+	RequireDockerVersion(t, ">=1.8.0")
```

Comment:
This should be `RequireDockerVersion(t, ">=1.9.0")` because `tag` was added then: https://github.com/docker/docker/blob/master/CHANGELOG.md#logging-7/

We should also have a separate test where `RequireDockerVersion(t, "<1.12.0,>=1.8.0")` that uses `fluentd-tag` for the old option.